lib/Parser.js

```diff
@@ -234,8 +234,8 @@ Parser.prototype.initializeEvaluating = function() {
 	this.plugin("evaluate MemberExpression", function(expression) {
 		var expr = expression;
 		var exprName = [];
-		while(expr.type === "MemberExpression" && !expr.computed) {
-			exprName.unshift(expr.property.name);
+		while(expr.type === "MemberExpression") {
+			exprName.unshift(expr.property.name || expr.property.value);
```

Comment:
`computed` exist for Literal and doesn't exist for Identifier ([esprima](http://esprima.org/demo/parse.html?code=a.b()%3B%0Aa%5B%27b%27%5D()%3B%0A)). I thought that exactly why parser didn't match to `a[b]` syntax